test-suite/exonum-py-tests/exonum_tests/deploy.py

```diff
@@ -0,0 +1,41 @@
+"""Tests for Exonum service deploy mechanism based on `exonum-launcher` tool."""
+
+import unittest
+import time
+
+from exonum_launcher.configuration import Configuration
+from exonum_launcher.launcher import Launcher
+
+from suite import run_dev_node, assert_processes_exited_successfully, launcher_networks
+
+
+class DeployTest(unittest.TestCase):
+    """Tests for Exonum deploy process."""
+
+    def test_deploy_run_dev(self):
+        """Tests the deploy mechanism in run-dev mode."""
+        with run_dev_node("exonum-cryptocurrency-advanced") as network:
+            # Wait some time for node to start
+            time.sleep(5)
```

Comment:
To be honest, I really don't like this approach (it's at the same time error-prone and not reliable), but I don't know if we have a correct way to check if node has completed initialization (including connecting to other peers and starting the consensus) and ready for interaction.

Reply:
We will be able to get a consensus status via `/healthcheck` when we will fix `ECR-2931`, won't we ?

Reply:
Yeah, I think so (however, test for healtcheck endpoint itself won't be able to use that method :upside_down_face:).

Reply:
Sure 😄 But for other tasks.